ui/perfherder/tests/TestsTable.jsx

```diff
@@ -87,7 +88,13 @@ export default function TestsTable(props) {
       accessor: 'total_alerts',
       Cell: (props) => {
         const { original } = props;
-        return <AlertsList alerts={original} />;
+        return (
+          <AlertsLink
+            alerts={original}
+            framework={framework}
+            frameworks={frameworks}
```

Comment:
I would rename frameworks to allFrameworks, framework and frameworks can be mixed up easily.